app/src/main/java/org/mozilla/rocket/content/ChromeViewModelFactory.kt

```diff
@@ -0,0 +1,29 @@
+package org.mozilla.rocket.content
```

Comment:
I'm not sure why we need ViewModelFactory if we don't have any tests

Reply:
I used to use the ViewModelFactory to pass the dependencies into the ViewModel. Or do you have any sugguestion?